tasks/setup_infrared.yml

```diff
@@ -8,7 +8,8 @@
 - name: Add all plugins
   shell: |
       git fetch "https://review.gerrithub.io/redhat-openstack/infrared" refs/changes/76/463576/2 && git checkout FETCH_HEAD
-      virtualenv .venv && source .venv/bin/activate
+      pip3 install virtualenv
```

Comment:
What if ansible host is python 2.7?

Reply:
We have stayed in our minimum requirements that we need python 3 right?